notebooks/quality_control.ipynb

```diff
@@ -0,0 +1,103 @@
+    "\n",
+    "Roundness is a useful metric in cell detection because it can help identify cells that have been improperly segmented. For example, cells that are elongated or irregular in shape may have been segmented incorrectly due to noise, overlapping cells, or other factors. By comparing the roundness of segmented cells to a threshold value, cells that are improperly segmented can be identified and corrected.\n",
+    "\n",
+    "### 2.2 Trace\n",
+    "**skewness (Stringer & Pachitariu, 2019):** The skewness of fluorescence activity from multi-day long experiments can be used as a quality control metric by providing an indication of the consistency and stability of the signal over time. A stable signal with consistent skewness values across multiple days suggests that the measurement system is working properly and the experimental conditions are consistent. However, changes in the skewness of fluorescence activity over time could indicate technical issues or changes in the sample physiology or behavior, which could affect the quality of the data and the accuracy of any conclusions drawn from the experiment.\n",
+    "\n",
+    "In vivo measurements may not necessarily show the same skewness as in vitro or ex vivo measurements, as the complexity and variability of the in vivo environment can introduce additional factors that may affect the fluorescence signal. For example, changes in the animal's behavior, stress levels, or physiological state could affect the skewness of the fluorescence activity over time. Therefore, it is important to establish appropriate baseline measurements and controls for in vivo experiments, and to carefully interpret any changes in skewness within the context of the experimental conditions and the animal's behavior and physiology.\n",
+    "\n",
+    "**variance (Stringer & Pachitariu, 2019):** The variance of fluorescence activity from multi-day long experiments can be used as a quality control metric by providing an indication of the consistency of the signal over time. A high variance could indicate that the signal is unstable, and there are large fluctuations in the fluorescence activity between different imaging sessions. On the other hand, a low variance suggests that the signal is stable and consistent over time.\n",
+    "\n",
+    "To use variance as a quality control metric, researchers can calculate the variance of the fluorescence activity across multiple days and compare it to a threshold value. If the variance exceeds the threshold, further investigation may be required to identify potential sources of instability in the imaging setup or data processing.\n",
```

Comment:
but are we computing fluorescence across multiple days?

Reply:
Not yet but there is no reason not to do it.

Reply:
Actually in this case, we calculate the skewness for each frame. I need to change the explanation accordingly.